src/spatialdata_plot/pl/basic.py

```diff
@@ -164,18 +166,18 @@ def render_shapes(
         cmap: Colormap | str | None = None,
         norm: bool | Normalize = False,
         scale: float | int = 1.0,
-        table_name: list[str] | str | None = None,
+        table_name: str | None = None,
         **kwargs: Any,
     ) -> sd.SpatialData:
         """
         Render shapes elements in SpatialData.
 
         Parameters
         ----------
-        elements : list[str] | str | None, optional
+        element : list[str] | str | None, optional
             The name(s) of the shapes element(s) to render. If `None`, all shapes
```

Comment:
Doesn't apply anymore. In general, the docstring needs to be updated in several places, right?

Reply:
yes, I will adjust the docstrings if overall the PR is accepted:) Just the type annotations already had to be changed for mypy

Reply:
Ah okay, perfect